build.gradle

```diff
@@ -71,3 +79,64 @@ dependencies {
         //implementation 'org.springframework:spring-web:5.0.2.RELEASE'
     }
 }
+
+def classFilesArray = []
+def execFilesArray = []
+
+task copyJavaClassFiles {
```

Comment:
Looks like this function copies both class files and execs. If so shall we rename the function to be self explanatory??

Reply:
I will rename it.

Reply:
Commit id is ff70c9ad45616a18a64eb54fda3c04bae55d0aa2